### DISCUSSION TITLE
Tolerance test - now what?

### DISCUSSION THREAD

**Discussion by tve** [OrcaSlicer/OrcaSlicer#1743] [Q&A] [answered]
I ran the orca tolerance test and the "0.2" comes out the best. What do I do with that info? Is this something I can enter somewhere or do I need to explicitly add that much to holes and other internal openings that need to have good fit?
BTW: why does this use a hex shape? It's quite tricky to measure accurately using calipers.

**Comment by SurgeHawk** [comment, marked as answer]
The tolerance test is mostly for your own information. It informs you of what kind of tolerances your printer is capable of. You could take that info and try adjusting your printers belt tension or other items on the physical printer, but 0.2 isn't bad. If you were designing parts it also tells you what kind of tolerances to design for.

**Comment by santiagomoneta** [comment]
is there something I should change on the X-Y hole and/or contour settings at the profile after this test?.. I don't design but want to be sure joints on models I download are OK

**Reply by Splinervision** [reply]
If you ever find an answer to that question I'd love to know it as well.  My tolerance is somewhere in the .15 range which I'm told is decent for FDM but doesn't help me much knowing that when I'm trying to print things that need accuracy.  I've wasted so much filament trying to print models that turn out to be worthless or won't fit together because they are way off.  Thing is, I can print a box or a square, adjust the X-Y Hole Settings and get it almost perfect for my own models, then load up someone else's model and that compensation doesn't seem to help.
